docs/reference/configuration/keywords.rst

```diff
@@ -2069,61 +2069,89 @@ Workflow hooks
 .. _hook_workflow:
 .. topic:: HOOK_WORKFLOW
 
-    With the keyword :code:`HOOK_WORKFLOW` you can configure workflow
-    'hooks'; meaning workflows which will be run automatically at
-    certain points during ERTs execution. Currently there are five
-    points in ERTs flow of execution where you can hook in a workflow,
-    before the simulations start, :code:`PRE_SIMULATION`; after all
-    the simulations have completed, :code:`POST_SIMULATION`; before the
-    update step, :code:`PRE_UPDATE`; :code:`POST_UPDATE`; after the update
-    step and :code:`PRE_FIRST_UPDATE` only before the first update.
-    :code:`PRE_FIRST_UPDATE` will run before :code:`PRE_UPDATE`.
-    For non iterative algorithms, :code:`PRE_FIRST_UPDATE` is equal to
-    :code:`PRE_UPDATE`. The :code:`POST_SIMULATION` hook is
-    typically used to trigger QC workflows:
+        With the keyword :code:`HOOK_WORKFLOW` you can configure workflow
+        'hooks'; meaning workflows which will be run automatically at certain
+        points during ERTs execution. Currently there are five points in ERTs
+        flow of execution where you can hook in a workflow:
 
-    ::
-
-        HOOK_WORKFLOW initWFLOW        PRE_SIMULATION
-        HOOK_WORKFLOW preUpdateWFLOW   PRE_UPDATE
-        HOOK_WORKFLOW postUpdateWFLOW  POST_UPDATE
-        HOOK_WORKFLOW QC_WFLOW1        POST_SIMULATION
-        HOOK_WORKFLOW QC_WFLOW2        POST_SIMULATION
+        - Before the simulations start using :code:`PRE_SIMULATION`,
```

Comment:
Should we point out that by "simulations" we mean "all forward models for a realization"?

Reply:
Yep! Updated!